File: CompetitiveProgramming/DWITE/DWITE '10 R1 #1 - Age Gate/2020-05-06/AgeGate.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
	cin.sync_with_stdio(0);
	cin.tie(0);
	
	for(int i = 0; i < 5; i++)
	{
		int d, m, y;
		cin >> d >> m >> y;

		if(y > 1997)
			cout << "too young" << endl;
		else if(y < 1997)
			cout << "old enough" << endl;
		else if(y == 1997)
		{
			if(m > 10)
				cout << "too young" << endl;
			else if(m < 10)
				cout << "old enough" << endl;
			else if(m == 10)
			{
				if(d > 27)
					cout << "too young" << endl;
				else if(d < 27)
					cout << "old enough" << endl;
				else if(d == 27)
					cout << "old enough" << endl;
			}
		}
	}
	
	return 0;
}

File: CompetitiveProgramming/DWITE/DWITE '11 R5 #1 - Age Gate/2020-12-29/AgeGate.py
for i in range(5):
	day, month, year = map(int, input().split())
	if year < 1997:
		print("old enough")
	elif year > 1997:
		print("too young")
	else:
		if month > 10:
			print("too young")
		elif month < 10:
			print("old enough")
		else:
			if day > 27:
				print("too young")
			else:
				print("old enough")